extensions/Worker.Extensions.Kafka/release_notes.md

```diff
@@ -0,0 +1,10 @@
+## Release notes
+<!-- Please add your release notes in the following format:
+- My change description (#PR/#issue)
+-->
+
+### 3.5.0
+- Added the improvement for the producer logging to app insights & filesystem. (https://github.com/Azure/azure-functions-kafka-extension/pull/377)
```

Comment:
@shrohilla Could you change the link to just the PR id prefixed with #. Ex (#377)
See the SDK release notes for reference: https://github.com/Azure/azure-functions-dotnet-worker/edit/main/sdk/release_notes.md

Reply:
not sure what exactly are you asking but it's not possible for cross repo and as per my understanding it looks fine to me.

<img width="541" alt="image" src="https://user-images.githubusercontent.com/80326489/184283005-06ee0a06-a3bd-4a63-aee9-1a56eb968073.png">